homeassistant/components/template/light.py

```diff
@@ -116,9 +122,13 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
             CONF_WHITE_VALUE_TEMPLATE: white_value_template,
         }
 
-        initialise_templates(hass, templates)
+        initialise_templates(hass, templates, attribute_templates)
```

Comment:
You could think about adding the new templates to `templates` instead. Maybe, if it's `extra_attributes`, it's more justified to keep it seprate.